FFTGeneratorConfigs.scala

```diff
@@ -0,0 +1,23 @@
+package fftgenerator
+
+import freechips.rocketchip.config.{Field, Parameters, Config}
+
+//cparameter to enable FFT
+case object FFTEnableKey extends Field[Option[FixedTailParams]](None)
+
+class WithFFTGenerator (enable: Boolean = true) extends Config((site, here, up) => {
+  case FFTEnableKey => Some(FixedTailParams())
+})
+
+// parameter to set FFT mmio registers base address
+case object FFTBaseAddrKey extends Field[Int](0x2000)
+
+class WithFFTBaseAddr (baseAddress: Int) extends Config((site, here, up) => {
+  case FFTBaseAddrKey => baseAddress
```

Comment:
I would make a new case class (say `FFTParams()`) that encompasses the `FixedTailParams` and the address so that you don't have an extra key/val pair used.

Reply:
Added baseAddress directly to FixedTailParams since FixedTailParams is already sufficiently removed from the FFT's case class.